go-controller/vendor/github.com/coreos/go-iptables/iptables/iptables.go

```diff
@@ -291,6 +373,11 @@ func (ipt *IPTables) Stats(table, chain string) ([][]string, error) {
 
 	ipv6 := ipt.proto == ProtocolIPv6
 
+	// Skip the warning if exist
```

Comment:
you mean the comments? I don't get this logic of cutting the 1st line..

Reply:
this isnt my code either, its just from the version bump. To see what I changed:
https://github.com/coreos/go-iptables/commit/077e672b3808a90d5be13111c722a73f50479153

Reply:
yikes! sorry for the noise tim, haha face palm!